### ISSUE TITLE
Setting date programmatically broken, clears input field.

### ISSUE THREAD

**Issue by rubencodes** [amsul/pickadate.js#830]
TL;DR: setting the date via the [documented API](http://amsul.ca/pickadate.js/api/#method-set-select) using a JS Date object and an array is broken. Possibly other methods too, but these were the ones I tried.

The method I'm referring to is: 

    //documented API for selecting date manually
    $('#datePicker').pickadate('set','select', [2016, 2, 8]);

Seems to work great in the demo, but I can't seem to replicate this behavior. In my attempts, the input field is cleared and the input value is reset as well. Am I doing something wrong here?

Here's a codepen demonstrating the behavior: 
http://codepen.io/anon/pen/zrMpmG

**Comment by rubencodes** [comment]
My bad, turned out to be using the API wrong. I was going based off of issue #356, but I see now the API has been updated to the format: 

    const picker = $('#datePicker').pickadate('picker');
    picker.set('select', [2016, 2, 8]);

This worked as expected!
